packages/dts-plugin/src/plugins/GenerateTypesPlugin.ts

```diff
@@ -88,10 +89,25 @@ export class GenerateTypesPlugin implements WebpackPluginInstance {
               return;
             }
             await generateTypesFn(finalOptions);
-
+            const config = finalOptions.remote.moduleFederationConfig;
+            let zipPrefix = '';
+            if (
+              typeof config.manifest === 'object' &&
+              'filePath' in config.manifest
+            ) {
+              zipPrefix = config.manifest.filePath;
+            } else if (
+              typeof config.manifest === 'object' &&
+              'fileName' in config.manifest
```

Comment:
additionally, is it ok that you check for fileName, but then you use filePath?

Reply:
good find!